packages/frontend/components/SoundBoard/SoundBoard.module.scss

```diff
@@ -1,12 +1,12 @@
-@import '../styles/variables';
+@import 'styles/variables';
 
 $progress-bar-height: 10px;
 
 .container {
   background: #2c2c2c;
   padding: 2em 2.5em;
   margin: 2em;
-  border-radius: 1.5em;
+  border-radius: 10px;
```

Comment:
Rounding looks strange particularly when zoomed out. Perhaps make it a percentage?

Reply:
I don't see anything strange but i've made the players scale better when zoomed out.